packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,148 @@ section: get-started
 import './get-started.css';
 import { Divider } from '@patternfly/react-core';
 
+
+## PatternFly 5.4
+The final minor release within PatternFly 5 is now live! Next up is PatternFly 6. Don't forget to keep an eye on [our staging site](https://staging-v6.patternfly.org/) for updates. 
+
+But first, we have *lots* to share for this update&mdash;check out the highlights.
+
+### Promoted package versions
+
+- need list
+
+### Component updates
+#### Fixed bugs
+
+There were still a handful of bugs on our to-do list. To address a variety of reported issues, we did the following:
+
+- Added support for additional label info on horizontal forms.
+- Added a missing border on compound expansion table rows.
+- Fixed `<ExpandableSection>` errors that were caused by `variant="truncate"`
+- Fixed issues with icon variants in modal.
+- Updated `paginationAriaLabel` to stop adding "items" to the end of labels.
+- Fixed unexpected wrapping in the pagination component. 
+- Added `shouldFocusFirstMenuItemOnOpen` to the dropdown component, to fix autofocus issues.
+- Fixed spacing issues with progress stepper's help text title.
+- Fixed some charts-related errors.
+- Added new `shouldFocusContent` property to the wizard component to help navigate accessibility challenges when navigating wizards with keyboard controls.
+- Improved accessibility features for the tree view component via aria properties.
+- Fixed an issue in the code block component that was removing code styling terms, like "\n" and "\t".
+- Updated the truncate component to only show a tooltip when truncation is present.
+- Drag and drop: 
+  - Switched to the flex layout for multi-list drag and drops.
```

Comment:
I would mention that this is a change to the example.  The layout is not defined by the component.